playbooks/update_files.yml

```diff
@@ -271,22 +359,30 @@
                 # nothing to do
                 echo No updates for role {{ inventory_hostname }}
                 exit 0
+              fi
+              commit_file={{ update_files_commit_file | quote }}
+              branch={{ update_files_branch | quote }}
+              if [ "{{ __is_new | ternary('true', 'false') }}" = true ]; then
+                git commit -s -F "$commit_file"
               else
-                git commit -s -F {{ update_files_commit_file | quote }}
-                force_flag="{{ __is_new | ternary('', '-f') }}"
-                if [ {{ lsr_dry_run | d('true') }} = false ]; then
-                  git push $force_flag -u origin {{ update_files_branch | quote }}
-                echo
-                  echo git push $force_flag -u origin {{ update_files_branch | quote }}
-                fi
+                # commit orig head, then amend, and ensure commit is signed
+                git commit -C ORIG_HEAD
```

Comment:
Why is this needed?

Reply:
This is needed so that you can edit existing PRs

Reply:
Right now I see, that's smart